src/clims/services/workflow/__init__.py

```diff
@@ -291,6 +308,7 @@ def get_tasks_by_ids(self, task_ids):
         task_ids_by_provider = defaultdict(list)
 
         for task_id in task_ids:
+            print(task_id, "HERE")
```

Comment:
This seems to be a debug line. Should it be removed?

Reply:
Yes :| thanks